### DISCUSSION TITLE
How can I ignore a file from being watched?

### DISCUSSION THREAD

**Discussion by seanstrom** [parcel-bundler/parcel#8736] [Q&A]
I want to be able to ignore a file from triggering a file watch event, but I don't know how to do that. I think parcel uses a file watcher that supports ignoring files, but is it possible to configure the watcher to ignore something through parcel?

**Comment by mischnic** [comment]
What kind of file is it? Parcel (as in `parcel`) should already only react to relevant file changes, even if the watcher (`@parcel/watcher`) sees more changes
